NCS.DSS.ContentPushService/PushService/MessagePushService.cs

```diff
@@ -87,13 +87,13 @@ public async Task PushToTouchpoint(string AppIdUri, string ClientUrl, BrokeredMe
             {
                 message.Complete();
                 await SaveNotificationToDBAsync((int)response.StatusCode, message.MessageId, notification, appIdUri, clientUrl, bearerToken, true);
-                log.LogInformation(string.Format("Saving to DB: Responsecode: {0} ResourceUrl: {1} MessageId: {2}", response?.StatusCode, notification.ResourceURL, message.MessageId));
```

Comment:
response could still be null? may be worth leaving it as (int) response?.StatusCode